utils/build_system/create_test_release_jobs.py

```diff
@@ -18,12 +24,12 @@ def create_directory(self, name, display_name):
             dir_xml_data = DIR_TEMPLATE % dict(sct_display_name=display_name)
             self.jenkins.create_job(f'{self.base_job_dir}/{name}', dir_xml_data)
         except jenkins.JenkinsException as ex:
-            print(ex)
+            LOGGER.info(ex)
```

Comment:
exception as INFO?

Reply:
The idea was just to log why the job or folder were not created. Usually it is because folder or job alread exists

Reply:
Added new method which allow to log jenkins exception as error or info